spec/features/user/user_edits_credentials_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
```

Comment:
I don't see anything about a platform admin here. I think this is related to a different story and shouldn't be in this PR.